server/models/event.js

```diff
@@ -0,0 +1,74 @@
+export default (sequelize, DataTypes) => {
+  const Event = sequelize.define('Event', {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notEmpty: {
+          args: true,
+          msg: 'Event name cannot be empty'
+        },
+        isAlphanumeric: {
+          args: true,
+          msg: 'Allows only alphanumeric characters'
+        }
+      },
+      set(value) {
```

Comment:
what is happening here, please 😳

Reply:
Thanks for pointing it out, I will remove this constraint